packages/babel-plugin-remove-graphql-queries/.babelrc

```diff
@@ -1,5 +1,5 @@
 {
   "presets": [
-    ["../../.babel-preset.js"]
+    ["gatsby"]
```

Comment:
OK so I _thought_ we were on the same page here, but this seems weird. I don't think we'd use the `gatsby` preset _for_ our own packages, but rather we'd expose this so that other gatsby _sites_ could use this in e.g. jest tests.

Reply:
Right... This makes a lot more sense now. 🤕

Reply:
💯% my fault in the issue. I can totally see how it could be interpreted both ways!

Reply:
Should we still use the same preset for our internal packages? I find it confusing that we have two different presets apparently 🙂

Reply:
What about `babel-preset-gatsby` (for sites _using_ gatsby) and `babel-preset-gatsby-package` (for packages/utilities for gatsby)

Also open to opinions here 🤷‍♂️

Reply:
I was more thinking about unifying those presets - Not sure if that's possible though. 🙂 Before we start adding an option `{ package: true }` it's probably better to separate those.

Our **packages** seem to use:

- [`@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env)
- [`@babel/preset-react`](https://babeljs.io/docs/en/babel-preset-react)
- [`@babel/preset-flow`](https://babeljs.io/docs/en/babel-preset-flow)
- [`@babel/plugin-proposal-class-properties`](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties)
- [`@babel/plugin-proposal-optional-chaining`](https://babeljs.io/docs/en/babel-plugin-proposal-optional-chaining)
- [`@babel/plugin-transform-runtime`](https://babeljs.io/docs/en/babel-plugin-transform-runtime#docsNav)

For **Jest config of sites** we seem to suggest:

```js
const babelOptions = {
  presets: ["@babel/react", "@babel/env"],
  plugins: [
    "@babel/plugin-proposal-optional-chaining",
    "@babel/plugin-proposal-class-properties",
  ],
}
```

Which can probably be the same as the one for **packages** without the browser stuff.

For **sites** (the thing we generate with babel-loader-helpers) the thing seems to be a lot more complex, involving different stages and something referred to as required/fallback plugins). In the Babel documentation article we mention this config:

```js
{
  presets: [
    [
      "@babel/preset-env",
      {
        loose: true,
        modules: false,
        useBuiltIns: "usage",
        shippedProposals: true,
        targets: {
          browsers: [">0.25%", "not dead"],
        },
      },
    ],
    [
      "@babel/preset-react",
      {
        useBuiltIns: true,
        pragma: "React.createElement",
      },
    ],
  ],
  plugins: [
    [
      "@babel/plugin-proposal-class-properties",
      {
        loose: true,
      },
    ],
    "@babel/plugin-syntax-dynamic-import",
    "babel-plugin-macros",
    [
      "@babel/plugin-transform-runtime",
      {
        helpers: true,
        regenerator: true,
      },
    ],
  ],
}
```

Know I realize why you were talking about doing this in a follow-up 🙈

Reply:
I _think_ it makes sense to remove the `fallbackPlugins`/`fallbackPresets` from the babel-loader in favor of the `babel-preset-gatsby` which should be the only fallback for **sites**. This means that we need:

- @babel/preset-env
- @babel/preset-react
- @babel/plugin-proposal-class-properties
- babel-plugin-macros
- @babel/plugin-syntax-dynamic-import
- @babel/plugin-transform-runtime

We probably don't want to add `babel-plugin-macros` and `@babel/plugin-syntax-dynamic-import` for our **packages** and also don't want to add `@babel/preset-flow` and `@babel/plugin-proposal-optional-chaining` to our sites. So I guess we should still separate for now or do you have an idea how we can unify this?